code/game/machinery/doors/blast_door.dm

```diff
@@ -123,15 +128,33 @@
 					usr << "<span class='warning'>You don't have enough sheets to repair this! You need at least [amt] sheets.</span>"
 
 
+		else if(src.density)
+			var/obj/item/weapon/W = C
+			user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN*2)
```

Comment:
Because of the fact that I wanted to strike a balance between the changeling arm_blade and the fact that these were hitherto impenetrable blast doors.

With the changeling arm blade, it takes 20 (600/(40/2)) swipes, meaning they can cleave through it in about 30 seconds, which is rather fast IMO for a blast-resistant door. Conversely doubling the number of strikes just makes it, again IMO, more tedious to break through, and spams comparatively more attack logs. The extra delay only affects attacking the doors, keeps the number of strikes the same and still pads out the amount of time it takes for them to break through the blast door, allowing someone attentive to respond to witness/call backup.